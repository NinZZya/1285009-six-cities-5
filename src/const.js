export const AppPath = {
  ROOT: `/`,
  LOGIN: `/login`,
  CITY: `/city`,
  OFFER: `/offer`,
  FAVORITES: `/favorites`,
  NOT_FOUND: `/not-found`,
};

export const CITIES = {
  1: {id: 1, name: `Paris`},
  2: {id: 2, name: `Cologne`},
  3: {id: 3, name: `Brussels`},
  4: {id: 4, name: `Amsterdam`},
  5: {id: 5, name: `Hamburg`},
  6: {id: 6, name: `Dusseldorf`},
};

export const SortType = {
  POPULAR: `Popular`,
  LOW_TO_HIGH: `Price: low to high`,
  HIGH_TO_LOW: `Price: high to low`,
  TOP_RATED_FIRST: `Top rated first`,
};

export const STARS = [
  {value: 5, title: `perfect`},
  {value: 4, title: `good`},
  {value: 3, title: `not bad`},
  {value: 2, title: `badly`},
  {value: 1, title: `terribly`},
];
