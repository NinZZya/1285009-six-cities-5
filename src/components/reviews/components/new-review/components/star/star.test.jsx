import React from 'react';
import renderer from 'react-test-renderer';
import Star from './star';
import {STARS} from '../../../../../../constants/const';

const star = STARS[1];
const testing = () => {};

it(`Should Star render correctly`, () => {
  const tree = renderer
    .create(
        <Star
          star ={star}
          onStarChange={testing}
        />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
