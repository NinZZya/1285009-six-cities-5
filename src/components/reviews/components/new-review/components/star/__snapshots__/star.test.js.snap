// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Star render correctly 1`] = `
Array [
  <input
    className="form__rating-input visually-hidden"
    id="4-stars"
    name="rating"
    onChange={[Function]}
    type="radio"
    value={4}
  />,
  <label
    className="reviews__rating-label form__rating-label"
    htmlFor="4-stars"
    title="good"
  >
    <svg
      className="form__star-image"
      height="33"
      width="37"
    >
      <use
        xlinkHref="#icon-star"
      />
    </svg>
  </label>,
]
`;
